.pre-commit-config.yaml

```diff
@@ -12,10 +12,3 @@ repos:
       rev: v1.43.0
       hooks:
       -   id: terraform_fmt
-  -   repo: https://github.com/markdownlint/markdownlint
```

Comment:
Why are we removing markdown lint?

Reply:
Unfortunately this markdown solution is incompatible with python3 and was causing tons of issues.  We've talked about finding a new linter, but haven't made it a priority